recipes/gulp/gulpfile.js

```diff
@@ -66,7 +67,7 @@ const handleOk = function(results) {
 const handleError = function(e) {
   stopServer();
   console.error(e); // eslint-disable-line no-console
-  throw e; // Throw to exit process with status 1.
+  process.exit(1);
```

Comment:
Just to be consistent with finishing run in `handleOk` and `handleError` methods